demos/code/code.js

```diff
@@ -259,8 +261,8 @@ Code.tabClick = function(clickedName) {
     try {
       xmlDom = Blockly.Xml.textToDom(xmlText);
     } catch (e) {
-      var q =
-          window.confirm(MSG['badXml'].replace('%1', e));
+      var q = window.confirm(
+          MSG['parseError'].replaceAll('%1', 'XML').replace('%2', e));
```

Comment:
`.replaceAll` (here and once below for JSON) doesn't exist in IE.  Since this code doesn't get transpiled, let's use `.replace(/%1/g, 'XML')` instead.